authors/apps/authentication/backends.py

```diff
@@ -8,66 +8,45 @@
     Authorization: token yourtokenhere
 """
 import jwt
-
 from django.conf import settings
-from django.http import HttpResponse
-from rest_framework import authentication, exceptions
-from rest_framework import status
-
+from rest_framework import authentication
+from rest_framework.exceptions import AuthenticationFailed
 from .models import User
 
 
 class JWTAuthentication(authentication.BaseAuthentication):
+
     def authenticate(self, request):
         """
             This method checks if the token is valid
-
+            the return value is the users token
         """
+
         auth = authentication.get_authorization_header(request).split()
+
+        # Ensure we have a token
         # get_authorization_header returns headers as bytesstring
         # hence the need to check
         # against b'token'
         if not auth or auth[0].lower() != b'token':
```

Comment:
Since Auth2.0 uses Bearer,  update this line to:
```suggestion
        if not auth or auth[0].lower() != b'bearer':
```

Reply:
Done